docs/sources/developers/plugins/plugin.schema.json

```diff
@@ -562,26 +569,103 @@
       }
     },
     "extensions": {
-      "type": "array",
-      "description": "List of link and component extensions which the plugin registers to given extension points.",
-      "items": {
-        "type": "object",
-        "properties": {
-          "extensionPointId": {
-            "type": "string"
-          },
-          "title": {
-            "type": "string"
-          },
-          "description": {
-            "type": "string"
-          },
-          "type": {
-            "type": "string",
-            "enum": ["link", "component"]
+      "type": "object",
+      "description": "...",
+      "properties": {
+        "addedComponents": {
+          "type": "array",
+          "description": "...",
+          "items": {
+            "type": "object",
+            "properties": {
+              "targets": {
+                "type": "array",
+                "description": "...",
+                "items": {
+                  "type": "string"
```

Comment:
Do you think it would be worth defining a pattern for the extension point id?

Reply:
Yeah, I was also thinking about something like this, I just couldn't come up with a "good pattern". (It would be nice if it could validate that it is prefixed with the plugins id, but I don't think that's possible.) A basic pattern would be good though!

Reply:
Addressed it in https://github.com/grafana/grafana/pull/92732/commits/0a7ff9c5039f686bd0b912aa0f8d71a35c98a22d.